### PR TITLE
Transfer Tokens to brave wallets

### PR DESCRIPTION
**Pull request by husobee**
### Summary

Allows linking and draining of suggestions from one brave wallet to another.

### Type of change ( select one )

- [x] Product feature
- [ ] Bug fix
- [ ] Performance improvement
- [ ] Refactor
- [ ] Other

### Tested Environments

- [ ] Development
- [ ] Staging
- [ ] Production

### Before submitting this PR:

- [ ] Does your code build cleanly without any errors or warnings?
- [ ] Have you used auto closing keywords?
- [ ] Have you added tests for new functionality?
- [ ] Have validated query efficiency for new database queries?
- [ ] Have documented new functionality in README or in comments?
- [ ] Have you squashed all intermediate commits?
- [ ] Is there a clear title that explains what the PR does?
- [ ] Have you used intuitive function, variable and other naming?
- [ ] Have you requested security / privacy review if needed
- [ ] Have you performed a self review of this PR?

### CODE REVIEW

File: wallet/datastore.go [hunk 1]
```diff
@@ -364,6 +364,7 @@ func (pg *Postgres) LinkWallet(ID string, userDepositDestination string, provide
 
 	err = pg.TxLinkWalletInfo(tx, ID, userDepositDestination, providerLinkingID, anonymousAddress, depositProvider)
 	if err != nil {
+		fmt.Println("!!!! err: ", err)
```
Comment (maikelmclauflin) [inline wallet/datastore.go:367]: we can remove this no?

File: wallet/service.go [hunk 2]
```diff
@@ -195,6 +199,19 @@ func SetupService(ctx context.Context, r *chi.Mux) (*chi.Mux, context.Context, *
 	if err != nil {
 		logger.Fatal().Err(err).Msg("failed to initialize wallet service")
 	}
+
+	// setup reputation client
+	var reputationClient reputation.Client
+	if os.Getenv("ENV") != "local" || len(os.Getenv("REPUTATION_SERVER")) > 0 {
```
Comment (maikelmclauflin) [inline wallet/service.go:205]: i believe these checks are already in the new fn

Comment (husobee) [reply wallet/service.go]: updated in https://github.com/brave-intl/bat-go/pull/541/commits/4a6999e3d9af426bc5ac753a8fa816662c03bea0

File: wallet/datastore.go (hunk 1 above)
Comment (husobee) [reply wallet/datastore.go]: updated in https://github.com/brave-intl/bat-go/pull/541/commits/4a6999e3d9af426bc5ac753a8fa816662c03bea0

**Review by maikelmclauflin — APPROVED** [review]

**Pull request merged**